aws-ts-appsync/iam.ts

```diff
@@ -14,17 +14,17 @@ export function createIamRole(name: string, table: aws.dynamodb.Table) {
                     }],
                     effect: "Allow",
                 }],
-            }, { async: true }).then(doc => doc.json),
+            }, { async: true }).json,
```

Comment:
We don't need the `async: true` invoke option, right? (Applies throughout)

```suggestion
            }).json,
```

Reply:
Love this comment, thank you! Turns out

```
Invoking data source functions
     * synchronously is deprecated.  The ability to do this will be removed at a later point in
     * time.
```

Definitely need to remove.